components/info_box/ActionBox.tsx

```diff
@@ -66,6 +72,12 @@ const ActionBox = ({
       formRef.current &&
       !formRef.current.email.validity.typeMismatch
     ) {
+      //TODO make this more robust (only counting when we should?)
+      fetch(
+        "../api/signup_count/" + constituencyData.constituencyIdentifiers.slug,
+        { method: "POST" },
+      ); // may as well count them now.
```

Comment:
Shift this into the `if (anResponse.ok)` block below?

I'm half-wondering long-term if we want to shift the whole signup server-side, so we have the ActionNetwork API call and the updating signup count in a single server-side action - makes it harder for a scripter to manipulate the numbers (obviously still possible, but just raises the bar ever so slightly / makes it slightly less obvious from a simple inspection of the browser network tab how to do it...!)

Reply:
I thought given these are just best-effort and will be updated later if it goes wrong better to let them run concurrently to make the UI faster for the user.